test/e2e/scheduling/limit_range.go

```diff
@@ -222,6 +226,103 @@ var _ = SIGDescribe("LimitRange", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should list, patch and delete a LimitRange by collection", func() {
+
+		ns := f.Namespace.Name
+		lrClient := f.ClientSet.CoreV1().LimitRanges(ns)
+		lrName := "e2e-limitrange-" + utilrand.String(5)
+		e2eLabelSelector := "e2e-test=" + lrName
+		patchedLabelSelector := lrName + "=patched"
+
+		min := getResourceList("50m", "100Mi", "100Gi")
+		max := getResourceList("500m", "500Mi", "500Gi")
+		defaultLimit := getResourceList("500m", "500Mi", "500Gi")
+		defaultRequest := getResourceList("100m", "200Mi", "200Gi")
+		maxLimitRequestRatio := v1.ResourceList{}
+
+		limitRange := &v1.LimitRange{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: lrName,
+				Labels: map[string]string{
+					"e2e-test": lrName,
+					lrName:     "created",
+				},
+			},
+			Spec: v1.LimitRangeSpec{
+				Limits: []v1.LimitRangeItem{
+					{
+						Type:                 v1.LimitTypeContainer,
+						Min:                  min,
+						Max:                  max,
+						Default:              defaultLimit,
+						DefaultRequest:       defaultRequest,
+						MaxLimitRequestRatio: maxLimitRequestRatio,
+					},
+				},
+			},
+		}
+		// Create a copy to be used in a second namespace
+		limitRange2 := &v1.LimitRange{}
+		*limitRange2 = *limitRange
+
+		ginkgo.By(fmt.Sprintf("Creating LimitRange %q in namespace %q", lrName, f.Namespace.Name))
+		limitRange, err := lrClient.Create(context.TODO(), limitRange, metav1.CreateOptions{})
```

Comment:
create a context at the beginning that you can use in all the calls.

Reply:
Fixed